superdesk/storage/amazon/amazon_media_storage.py

```diff
@@ -286,3 +305,45 @@ def remove_unreferenced_files(self, existing_files):
                 print('Failed to clean orphans: {}'.format(errors))
         else:
             print('There\'s nothing to clean.')
+
+    def find(self, prefix=None, upload_date=None, resource=None):
```

Comment:
imo we can call this `prefix` param `folder` as well, it's supposed to use same value right?